Original message by Sergio Durigan Junior

  <https://sourceware.org/ml/gdb-patches/2013-09/msg00547.html>
  Message-ID: <m37gefntd6.fsf@redhat.com>

====

commit e83b28bc76d98e8b1f853ad5d72028c682695a37
Author: Sergio Durigan Junior <sergiodj@redhat.com>
Date:   Tue Sep 17 19:24:51 2013 +0000

    This commit fixes a regression introduced by the new $_isvoid convenience
    function.  It adds a check for $_isvoid during the test of "show convenience"
    output.
    
    gdb/testsuite/
    2013-09-17  Sergio Durigan Junior  <sergiodj@redhat.com>
    
    	* gdb.base/defaults.exp (<show_conv_list>): Add check for $_isvoid
    	convenience function.

Index: gdb-7.2/gdb/testsuite/gdb.base/default.exp
===================================================================
--- gdb-7.2.orig/gdb/testsuite/gdb.base/default.exp
+++ gdb-7.2/gdb/testsuite/gdb.base/default.exp
@@ -589,6 +589,7 @@ gdb_test_list_exact "show convenience" "
 	{$_probe_arg7 = <error: No frame selected>} \
 	{$_probe_arg8 = <error: No frame selected>} \
 	{$_probe_arg9 = <error: No frame selected>} \
+	{$_isvoid = <internal function _isvoid>} \
     }
 
 #test show directories
